cmd/root.go

```diff
@@ -92,6 +94,12 @@ Welcome to the Astro CLI, the modern command line interface for data orchestrati
 		newAuthCommand(),
 	)
 
+	if config.CFG.SQLCLI.GetBool() {
+		rootCmd.AddCommand(
+			sql.NewFlowCommand(),
+		)
+	}
```

Comment:
you can find missing tests by these messages from codecov ^^

Reply:
Yes working on those already. Completely missed the other files, sorry.